pkg/virt-operator/resource/apply/patches_test.go

```diff
@@ -102,6 +120,43 @@ var _ = Describe("Patches", func() {
 		})
 	})
 
+	Context("generically apply patches", func() {
```

Comment:
You don't need to create the `Context("generically apply patches", func() {` 3 times, you can have the 3 different `It`s in the first context